EasyLearn/app/assets/stylesheets/layout/cards_classes.css

```diff
@@ -1,22 +1,28 @@
 .cards-container {
   padding: 20px 60px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
```

Comment:
sort this block and delete the extra line

Reply:
Done.